app/utils/protobuf_util.py

```diff
@@ -19,6 +19,6 @@ def serialize_to_object(proto, descriptor = None):
             return result
 
     @staticmethod
-    def stream_file(f:FileStorage, size:int=100*1024*1024) -> bytes:
+    def stream_file(f:FileStorage, size:int=4190000) -> bytes:
```

Comment:
nit: this value looks arbitrary...

Reply:
Upper transference limit is 4MB, so we cannot occupy 4MB for file transference. The rest band is for head info and so on.

Reply:
Yes. 
I didn't calculate an edge case. I believe that about `4KB` is enough for the other info.